apps/api/src/__tests__/snips/v2/robots-txt.test.ts

```diff
@@ -0,0 +1,72 @@
+import { createRobotsChecker, fetchRobotsTxt } from "../../../lib/robots-txt";
+import { scrapeTimeout, idmux, Identity } from "./lib";
+
+let identity: Identity;
+
+beforeAll(async () => {
+  identity = await idmux({
+    name: "robots-txt",
+    concurrency: 100,
+    credits: 1000000,
+  });
+}, 10000);
```

Comment:
Please remove the `idmux` call -- since you don't call the API here, you don't need an Identity